airbyte-cdk/bulk/core/base/src/main/kotlin/io/airbyte/cdk/AirbyteConnectorRunner.kt

```diff
@@ -17,6 +17,11 @@ import picocli.CommandLine.Model.ArgGroupSpec
 import picocli.CommandLine.Model.OptionSpec
 import picocli.CommandLine.Model.UsageMessageSpec
 
+// A pair of micronaut environments, so that connectors can override micronaut properties
+// without needing to copy an entire application.yaml out of the core CDK
+const val OVERRIDE_ENV = "override"
+const val TEST_OVERRIDE_ENV = "test-override"
```

Comment:
Is this meant to be `test`?  I didn't see any `application-test-override.yaml` files in the PR.  Is this just for future support?

Reply:
I should just delete this actually. Had a theory that connectors might want to only override part of the application-test.yaml, at which point we'd need yet another env

but app-test only contains a single property, so :shrug:

Reply:
done in https://github.com/airbytehq/airbyte/pull/51600/commits/c18f1a300fa022dac95b87fd705713a2828f6370